tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -272,78 +271,78 @@ def test_dispatch_endpoint_clients_can_connect_and_monitor_can_shut_down_evaluat
             assert snapshot.get_job("1", "1").status == FORWARD_MODEL_STATE_FAILURE
 
             # one monitor requests that server exit
-            monitor.signal_cancel()
+            await monitor.signal_cancel()
 
             # both monitors should get a terminated event
-            terminated = next(events)
-            terminated2 = next(events2)
+            terminated = await events.__anext__()
+            terminated2 = await events2.__anext__()
             assert terminated["type"] == identifiers.EVTYPE_EE_TERMINATED
             assert terminated2["type"] == identifiers.EVTYPE_EE_TERMINATED
 
-            for e in [events, events2]:
-                for undexpected_event in e:
-                    raise AssertionError(
-                        f"got unexpected event {undexpected_event} from monitor"
-                    )
+            # async for e in [events, events2]:
+            #     for undexpected_event in e:
+            #         raise AssertionError(
+            #             f"got unexpected event {undexpected_event} from monitor"
+            #         )
```

Comment:
Why are we no longer asserting this?

Reply:
Yes, that's a good question! I found it redundant, but can inject it back.

Reply:
Done!